truthbeam/processor.go

```diff
@@ -45,14 +46,32 @@ func (t *truthBeamProcessor) processLogs(ctx context.Context, ld plog.Logs) (plo
 		for j := 0; j < ilss.Len(); j++ {
 			ils := ilss.At(j)
 			logs := ils.LogRecords()
-			resource := rs.Resource()
 			for k := 0; k < logs.Len(); k++ {
 				logRecord := logs.At(k)
-				err := client.ApplyAttributes(ctx, t.client, t.config.ClientConfig.Endpoint, resource, logRecord)
+
```

Comment:
Not introduced by this PR, but this loop includes many variable names no so intuitive, such as `rl`, `rs`, `ld`, `ils`, `ilss`. Together I think they may confuse and decrease the readability. I had to reread some times to confirm what were they referring to. :) WDYT?

Reply:
I'll take a look. If it is a little refactor, I don't mind adding it in it's own commit here.